### PR TITLE
Parametrize log stream names to reduce cloudwatch throttling

### PR DESCRIPTION
**Pull request by QuentinBisson**
Tested on anteater (only place I could)

Towards https://github.com/giantswarm/giantswarm/issues/22993

```
time="2022-08-03T15:29:58Z" level=info msg="[cloudwatch 3] Log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-giantswarm/encryption-provider-operator does not exist in log group anteater-control-plane"
time="2022-08-03T15:29:58Z" level=info msg="[cloudwatch 3] Created log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-giantswarm/encryption-provider-operator in group anteater-control-plane"
time="2022-08-03T15:29:59Z" level=info msg="[cloudwatch 3] Log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-kube-system/k8s-api-server does not exist in log group anteater-control-plane"
time="2022-08-03T15:29:59Z" level=info msg="[cloudwatch 3] Created log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-kube-system/k8s-api-server in group anteater-control-plane"
time="2022-08-03T15:30:04Z" level=info msg="[cloudwatch 3] Log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-giantswarm/kyverno does not exist in log group anteater-control-plane"
time="2022-08-03T15:30:04Z" level=info msg="[cloudwatch 3] Created log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-giantswarm/kyverno in group anteater-control-plane"
time="2022-08-03T15:30:06Z" level=info msg="[cloudwatch 3] Log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-monitoring/net-exporter does not exist in log group anteater-control-plane"
time="2022-08-03T15:30:06Z" level=info msg="[cloudwatch 3] Created log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-monitoring/net-exporter in group anteater-control-plane"
time="2022-08-03T15:30:07Z" level=info msg="[cloudwatch 3] Log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-monitoring/promxy-app does not exist in log group anteater-control-plane"
time="2022-08-03T15:30:07Z" level=info msg="[cloudwatch 3] Created log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-monitoring/promxy-app in group anteater-control-plane"
time="2022-08-03T15:30:11Z" level=info msg="[cloudwatch 3] Log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-monitoring/silence-operator does not exist in log group anteater-control-plane"
time="2022-08-03T15:30:11Z" level=info msg="[cloudwatch 3] Created log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-monitoring/silence-operator in group anteater-control-plane"
time="2022-08-03T15:30:11Z" level=info msg="[cloudwatch 3] Log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-giantswarm/starboard-exporter does not exist in log group anteater-control-plane"
time="2022-08-03T15:30:11Z" level=info msg="[cloudwatch 3] Created log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-giantswarm/starboard-exporter in group anteater-control-plane"
time="2022-08-03T15:30:11Z" level=info msg="[cloudwatch 3] Log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-giantswarm/tokend does not exist in log group anteater-control-plane"
time="2022-08-03T15:30:11Z" level=info msg="[cloudwatch 3] Created log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-giantswarm/tokend in group anteater-control-plane"
time="2022-08-03T15:30:15Z" level=info msg="[cloudwatch 3] Log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-giantswarm/app-admission-controller does not exist in log group anteater-control-plane"
time="2022-08-03T15:30:15Z" level=info msg="[cloudwatch 3] Created log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-giantswarm/app-admission-controller in group anteater-control-plane"
time="2022-08-03T15:30:15Z" level=info msg="[cloudwatch 3] Log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-monitoring/fluent-bit does not exist in log group anteater-control-plane"
time="2022-08-03T15:30:15Z" level=info msg="[cloudwatch 3] Created log stream control-plane-ip-172-19-5-221.eu-west-1.compute.internal-containers-monitoring/fluent-bit in group anteater-control-plane"
```

### CODE REVIEW

**Comment by TheoBrigitte** [comment]
> Creation of the dockerfile can be found here: https://github.com/giantswarm/retagger/blob/ac59628bd44febdaccc695343510554e023a6fe1/images.yaml#L302

I cannot find the PR for retagger, did you just skip it ?

**Comment by QuentinBisson** [comment]
Yes :( I wanted to try things first because I never got how I could try retagger runs for testing purposes.
Do you want me to revert so I can craft a PR ?

**Comment by QuentinBisson** [comment]
https://github.com/giantswarm/retagger/pull/717

**Comment by TheoBrigitte** [comment]
> Yes :( I wanted to try things first because I never got how I could try retagger runs for testing purposes. Do you want me to revert so I can craft a PR ?

I know we cannot test with retagger, but now we cannot review what was done easily. I did found out what was done, so no need to revert what you did. We should open a PM/issue to ask for improvements in retagger.

File: helm/fluent-logshipping-app/values.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ project:
   commit: "[[ .SHA ]]"
 
 registry:
-  domain: docker.io
+  domain: quay.io
```
Comment (hervenicol) [inline helm/fluent-logshipping-app/values.yaml:14]: How is this related? Shouldn't the image be on both quay and docker?

**Review by TheoBrigitte — APPROVED** [review]
LGTM

**Review by hervenicol — APPROVED** [review]
Already live-discussed together, I like it.
LGTM if tested.

File: helm/fluent-logshipping-app/values.yaml (hunk 1 above)
Comment (QuentinBisson) [reply helm/fluent-logshipping-app/values.yaml]: It's not, i had to do it during testing because I cannot delete tags on docker
